### ISSUE TITLE
Forge not recognizing mod!

### ISSUE THREAD

**Issue by Zevidrex** [scalda/FoolCraft#43]
I know what you're thinking. You're probably thinking "hey buddy.. theres already a discussion about this". But thats not entirely true. You see, the other discussion mentioned using the twitch app to get minecraft mods. Theres just one tiny problem: I havent found a linux version of this app. How do I proceed? Should I cast my computer into a fiery abyss? Should I spill the blood of the ancients and hope a dark disturbing ritual gets foolcraft working on my computer? 

tl;dr, If forge is our only option, how do we forge to see the mod? Is a better way than using the dumb twitch app for babies? 
Hope to get this thing working. I'll update if the ritual works... or if i figure it out.

**Comment by scalda** [comment]
well you could always pull down the server files so you have the complete pack and then use multimc

**Comment by Zevidrex** [comment]
Very nice, except that when I try to create a new instance, It spits back about some instance.cfg that it doesnt have
